src/types.ts

```diff
@@ -0,0 +1,48 @@
+import { z } from 'zod';
+
+export const ChainExplorerAPIKeySchema = z.object({
```

Comment:
Nit: In other projects we use lowercase name for schemas with the `Schema` suffix (e.g. `chainExplorerAPIKeySchema`).